test/Webfactory/Slimdump/Config/ColumnTest.php

```diff
@@ -97,5 +97,16 @@ public function testReplaceColumnWithoutReplacement()
         $columnConfig = new Column($xmlElement);
         $this->assertEquals('', $columnConfig->processRowValue('test value'));
     }
+    public function testReplaceColumnWithUniqId()
+    {
+        $xml = '<?xml version="1.0" ?>
+                <column name="test" dump="replace" replacement="FAKER_unique->randomDigit" />';
+
+        $xmlElement = new \SimpleXMLElement($xml);
+
+        $columnConfig = new Column($xmlElement);
+        $this->assertRegExp('/[0-9a-zA-Z]*/', $columnConfig->processRowValue('test value'));
```

Comment:
Shouldn't randomDigit generate something like /[0-9]+/ - i.e. no non-numerical characters and at least one digit?